themes/courses.labster.com/lms/templates/main.html

```diff
@@ -0,0 +1,203 @@
+## coding=utf-8
+
+## This is the main Mako template that all page templates should include.
```

Comment:
Add up a comment that this file was copied from that place, and it's exactly the same.

And most important highlight a line in this file that makes it different from the original one.

Reply:
```
1 ### This is a copy of #file
2 The only difference is in line #10, 
...
8
9
10 # this is the only line different from original file
11 GOOGLE_TAG 
12 ....
```

Reply:
added

Reply:
Thank you, approved!